internal/cliutils/browser.go

```diff
@@ -0,0 +1,22 @@
+package cliutils
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+)
+
+func OpenInBrowser(url string) error {
```

Comment:
please also make the server use this function and delete the unnecessary implementation there